python/ray/rllib/ddpg/models.py

```diff
@@ -0,0 +1,185 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+from ray.rllib.models.catalog import ModelCatalog
+import tensorflow.contrib.layers as layers
+from ray.rllib.ddpg.ou_noise import AdaptiveParamNoiseSpec
+
+
+def _huber_loss(x, delta=1.0):
```

Comment:
Some of this seems duplicated with DQN, we should probably move it to a util file.